webui/react/src/shared/utils/routes.ts

```diff
@@ -48,9 +48,6 @@ export const parseUrl = (url: string): URL => {
 const stripUrl = (aUrl: string): string => {
   const url = parseUrl(aUrl);
   const rest = url.href.replace(url.origin, '');
-  if (rest.startsWith(process.env.PUBLIC_URL)) {
```

Comment:
why were we using this 🤔

Reply:
not 100% sure, but its for netlify and production mode. In local, there's `/det` but on netlify there isn't.
@trentwatt might know about it more.

Reply:
most likely I wrote this.. do we need to change or remove it?

Reply:
the `history.ts` is changed, so the basename is not included anymore. That's why I removed `replace()` part.

Reply:
thanks. I think if [these](https://github.com/determined-ai/determined/pull/4619/files#r929350828) use cases still work we should be good here